.github/workflows/ci.yml

```diff
@@ -293,7 +293,7 @@ jobs:
           cargo update -p tokio --precise 1.29.1
           cargo update -p tokio-util --precise 0.7.11
           cargo update -p idna_adapter --precise 1.1.0
-          cargo update -p hashbrown@0.15.2 --precise 0.15.0
```

Comment:
I'm guessing we don't need to pin hashbrown anymore?

Reply:
I haven't checked if it's changed, but this was because 2 versions of hashbrown were depended on by dev-dependencies.

Reply:
I tried another way in https://github.com/seanmonstar/reqwest/pull/2668 but doesn't work.

Hi @linyihai, would you like to try remove this line directly?

Reply:
I'd try it in the next commit.  😃

Reply:
Ooops, it doesn't work. I believe https://github.com/seanmonstar/reqwest/pull/2666/commits/1b5ed25e8231c9a3377845b07b9666ab36a57d8d is good enough. Thank you @linyihai for this.